cockatrice/src/dlg_connect.cpp

```diff
@@ -92,17 +112,16 @@ DlgConnect::DlgConnect(QWidget *parent)
     QGroupBox *loginGroupBox = new QGroupBox(tr("Login"));
     loginGroupBox->setLayout(loginLayout);
 
+	QGroupBox *btnGroupBox = new QGroupBox(tr(""));
```

Comment:
Any reason you give it an empty name? I think you can just do `new QGroupBox()` and be done with it